MapboxDirections/MBRouteStep.swift

```diff
@@ -668,6 +673,7 @@ open class RouteStep: NSObject, NSSecureCoding {
         coder.encode(codes, forKey: "codes")
         coder.encode(destinationCodes, forKey: "destinationCodes")
         coder.encode(destinations, forKey: "destinations")
+        coder.encode(voiceInstructions, forKey: "voiceInstructions")
```

Comment:
Remember to add something about `voiceInstructions` to `RouteStepTests.testCoding()`.